### ISSUE TITLE
 Window mode

### ISSUE THREAD

**Issue by azraelim94** [zzsort/Aion-Version-Dll#7]
From time to time, the mouse fix in window mode stops working, you need to restart the client, everything is fine with other modes

**Comment by zzsort** [comment]
Can you see if switching to fullscreen and back fixes it? I'll look for the issue.

**Comment by azraelim94** [comment]
No, I tried, it does not help. Need to turn off the game and turn it on again

**Comment by zzsort** [comment]
I think I hit this issue once after playing for about 10 minutes, but I couldn't get it to happen again. It seemed like the mouse fix was partially working (working better than without it), but I couldn't turn the camera to the right and it felt like the camera was lagging behind the mouse. 

It will help me fix it if I can narrow down possible causes, especially if there is a consistent way to do it. So if you have any ideas let me know. 

For me it started after I teleported to another world, and interacted with broker npcs, all while moving the camera around randomly and resizing the window. My ping on my lan was over 800ms, so I installed the TCP no delay fix, rebooted, ping went down to 16-32, and couldn't get it to happen again (played for a couple hours). The TCP fix is probably unrelated, but at this point I don't know.

**Comment by azraelim94** [comment]
Regarding the reproduction of the problem, I will unsubscribe as soon as I find a way. There is another problem - if you run while holding two mouse buttons, then release the left button, rotate the camera, the mouse will be inverted until both buttons are released

**Comment by flroexus** [comment]
Same issue

**Comment by Jonathan57500** [comment]
Same issue here. Looks to be an app crash, can we inject some logs into a client log file ?
If for any reason this script returns an exception, can you catch it and inform player in any way ?
Thanks for the great work, happy to see people that still share their work into the Internet ;)

**Comment by Jonathan57500** [comment]
It is working as expected when full mode screen, you will have this issue only in windowed mode

**Comment by zzsort** [comment]
I have a new fix for windowed mode, will try to get it uploaded in a few hours

**Comment by zzsort** [comment]
Please try 1.5 and let me know how it works

**Comment by flroexus** [comment]
thanks will try

**Comment by Jonathan57500** [comment]
I'll test in next hour, will let you know, anyway thank you so much for the fix, you tool is so usefull for me ;)

**Comment by flroexus** [comment]
How do I make a 64bit version.dll?

**Comment by zzsort** [comment]
This works for me, in the detours directory:
set DETOURS_TARGET_PROCESSOR=X64
call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
nmake

then set DETOURS_PATH and open the .sln and x64 should build

**Comment by Jonathan57500** [comment]
After new test: it's working PERFECTLY in windowed mode!!!
Thank you so much for your work :) :) :)
With an EU 2.7 client, this is what I tested that made this dll to crash before:
- Make a teleportation on any other map and right click before end of the map loading, before it crashed in windowed mode, and now it's working!
- Click on both left and right mouse mutton make dll stop working when keeping the right click on in windowed mode, now it's working perfectly !
Thanks again for your work ;)
